pyscf/adc/radc.py

```diff
@@ -94,47 +99,70 @@ def compute_amplitudes(myadc, eris):
 
     eris_oooo = eris.oooo
     eris_ovoo = eris.ovoo
-    eris_ovov = eris.ovov
     eris_oovv = eris.oovv
     eris_ovvo = eris.ovvo
-    eris_ovvv = eris.ovvv
 
-    e = myadc.mo_energy
+    # Compute first-order doubles t2 (tijab)
 
-    d_ij = e[:nocc][:,None] + e[:nocc]
+    v2e_oovv = eris_ovvo[:].transpose(0,3,1,2).copy()
 
+    e = myadc.mo_energy
+    d_ij = e[:nocc][:,None] + e[:nocc]
     d_ab = e[nocc:][:,None] + e[nocc:]
 
     D2 = d_ij.reshape(-1,1) - d_ab.reshape(-1)
-
     D2 = D2.reshape((nocc,nocc,nvir,nvir))
 
     D1 = e[:nocc][:None].reshape(-1,1) - e[nocc:].reshape(-1)
     D1 = D1.reshape((nocc,nvir))
 
-    # Compute first-order doubles t2 (tijab)
-
-    v2e_oovv = eris_ovov[:].transpose(0,2,1,3).copy()
-
     t2_1 = v2e_oovv/D2
+    if not isinstance(eris.oooo, np.ndarray):
+        t2_1 = radc_ao2mo.write_dataset(t2_1)
+        
+    del (v2e_oovv)
+    del (D2)
+
+    cput0 = log.timer_debug1("Completed t2_1 amplitude calculation", *cput0)
 
     # Compute second-order singles t1 (tij)
 
-    t2_1_a = t2_1 - t2_1.transpose(1,0,2,3).copy()
+    if isinstance(eris.ovvv, type(None)):
+        chnk_size = radc_ao2mo.calculate_chunk_size(myadc)
+    else :
```

Comment:
```suggestion
    else:
```

You might also want to run the files through a formatter like YAPF to unify the code style.